components/ConfirmationModalWindow/ConfirmationModalWindow.js

```diff
@@ -33,19 +33,27 @@ const ConfirmationModal = styled.section`
   align-items: center;
   flex-direction: column;
   justify-content: center;
-  background-color: rgb(255, 255, 255);
+  width: 80%;
+  height: 40%;
+  background-color: var(--color-light-yellow);
   border-radius: 8px;
   padding: 20px;
-  border: 3px rgb(255, 0, 0) solid;
+  border: 2px rgb(255, 0, 0) solid;
+  width: 90;
```

Comment:
Here is a duplicate `width property`. Only one width declaration is necessary, and it should include a unit of measurement (% in this case for percentage). 

```suggestion
```